src/patternfly/sass-utilities/scss-variables.scss

```
@@ -332,3 +336,12 @@ $pf-global--spacer-map: (
   "3xl": "var(--pf-global--spacer--3xl)",
   "4xl": "var(--pf-global--spacer--4xl)"
 );
+
+// Modified grid breakpoints
+@if $pf-global--full-page-breakpoints {
+  $pf-global--breakpoint--sm: pf-offset-variable-value($pf-global--breakpoint--sm, pf-size-ppx($pf-global--sidebar--Width));
```

I think you can use `calc()` here, too? Would this work? https://codepen.io/mcoker/pen/RwyGWay

```suggestion
  $pf-global--breakpoint--sm: calc(#{$pf-global--breakpoint--sm} - #{$pf-global--sidebar--Width});
```

---

I could, but you had mentioned creating a reusable mixin/function here earlier

---

@mcoker `calc` doesn't work bc the units differ `px` vs `em`